cypress/integration/patch/patch-route/tasks-filters.js

```diff
@@ -19,20 +19,30 @@ describe("Tasks filters", function() {
     const variantInputValue = "lint";
     it("Updates url with input value and fetches tasks filtered by variant", () => {
       cy.get("[data-cy=variant-input]").type(variantInputValue);
-      locationHasUpdatedVariantParam(variantInputValue, "VARIANT");
+      locationHasUpdatedVariantParam(variantInputValue, "variant");
       filteredTasksAreFetched("variant", variantInputValue);
       cy.get("[data-cy=variant-input]").clear();
       locationHasUpdatedVariantParam(null);
     });
   });
+
+  describe("Task name input field", () => {
+    const taskNameInputValue = "test-cloud";
+    it("Updates url with input value and fetches tasks filtered by task name", () => {
+      cy.get("[data-cy=task-name-input]").type(taskNameInputValue);
+      locationHasUpdatedVariantParam(taskNameInputValue, "taskName");
```

Comment:
we should remove "Variant" from this function name since we are also using it for task name. i also think this could be useful fo other tests and could belong in the utils folder.